wiki/Forum/en.md

```diff
@@ -0,0 +1,18 @@
+---
+stub: true
+tags:
+  - forum
+---
+
+#Forum
+
+The forums are a place where the [osu!](/wiki/Disambiguation/osu!) community can discuss a variety of topics. 
+
+The [forum index](https://osu.ppy.sh/community/forums) is generalised under a hierarchy:
+Forum -> Category (e.g; osu!) -> Section (e.g; development) -> Subforums (e.g; completed projects) -> Individual threads
+
+One of ther main functions of the forums is to assist with [beatmap moderation](/wiki/Modding) where users get feedback and help to improve their beatmaps. 
```

Comment:
this is not true anymore for a few years at least, all modding takes place in beatmap discussions